dev-client/src/screens/SoilScreen/utils/renderValues.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {TFunction} from 'i18next';
+import {
+  DepthDependentSoilData,
+  LabelledDepthInterval,
+  SoilPitMethod,
+} from 'terraso-client-shared/soilId/soilIdSlice';
+
+export const renderDepthInterval = ({
+  label,
+  depthInterval: {start, end},
+}: LabelledDepthInterval) => {
+  return `${label ? `${label}: ` : ''}${start}-${end} cm`;
```

Comment:
question: does this string need to be localized? this will probably break for RTL.

Reply:
yes, it also needs to be localized for units